blaze/compute/tests/test_numpy_compute.py

```diff
@@ -438,3 +438,24 @@ def test_nunique_recarray():
     s = symbol('s', discover(b))
     expr = s.nunique()
     assert compute(expr, b) == len(np.unique(b))
+
+
+def test_str_concat():
+    a = np.array(('a', 'b', 'c'))
+    s = symbol('s', discover(a))
+    expr = s.concat('a')
+    assert all(compute(expr, a) == np.char.add(a, 'a'))
+
+
+def test_str_repeat():
+    a = np.array(('a', 'b', 'c'))
+    s = symbol('s', discover(a))
```

Comment:
when you write these tests for `Series`, you'll need to explicitly create the symbol with a `string` dtype, else the methods won't show up on the symbol.

`discover(series)` when `series` has strings will produce a datashape with `object` measure. We should probably use pandas own type inference here to check whether a `Series` is actually object vs string